skada/_dasvm.py

```diff
@@ -0,0 +1,225 @@
+# Author: Ruben Bueno <ruben.bueno@polytechnique.edu>
+# dasvm implementation
+
+import numpy as np
+import math
+
+from skada.base import BaseEstimator
+from skada.utils import check_X_y_domain, source_target_split
+from sklearn.base import clone
+
+from sklearn.svm import SVC
+
+
+class DASVMEstimator(BaseEstimator):
```

Comment:
Maybe we can wait to have the PR of Rémi to have the BaseDAEstimator? wdyt @kachayev @rflamary